README.md

```diff
@@ -4,6 +4,301 @@
 
 JetStream is a fast library for LLM inference and serving on TPUs (and GPUs in future -- PRs welcome).
 
+## Documentation
+
+- [Online Inference with MaxText on v5e Cloud TPU VM](#jetstream-maxtext-inference-on-v5e-cloud-tpu-vm-user-guide)
+- [JetStream Standalone Local Setup](#jetstream-standalone-local-setup)
+
+---
+
+# JetStream MaxText Inference on v5e Cloud TPU VM User Guide
+
+## Outline
+
+
+1. Prerequisites: Prepare your GCP project and connect to Cloud TPU VM
+2. Download the JetStream and MaxText github repository
+3. Setup your MaxText JetStream environment
+4. Convert Model Checkpoints
+5. Run the JetStream MaxText server
+6. Send a test request to the JetStream MaxText server
+7. Run benchmarks with the JetStream MaxText server
+8. Clean up
+
+
+## Prerequisites: Prepare your GCP project and connect to Cloud TPU VM
+
+Follow the steps in [Manage TPU resources | Google Cloud](https://cloud.google.com/tpu/docs/managing-tpus-tpu-vm) to create a Cloud TPU VM (Recommend TPU type: `v5litepod-8`) and connect to the Cloud TPU VM.
+
+
+## Step 1: Download JetStream and the MaxText github repository
+
+```bash
+git clone -b jetstream-v0.2.0 https://github.com/google/maxtext.git
```

Comment:
Will this need to be updated with every jetstream release?

Reply:
For this user guide demo, we want to use the stable release version. Later we can update it or keep it as it is.

Reply:
Got it, thanks!